package.json

```diff
@@ -45,52 +46,42 @@
     "Vuex",
     "Vue"
   ],
-  "author": "Carlo Beltrame",
+  "authors": [
+    "Carlo Beltrame",
+    "Urban Suppiger"
+  ],
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/ecamp/hal-json-vuex/issues"
   },
   "homepage": "https://github.com/ecamp/hal-json-vuex#readme",
   "dependencies": {
     "hal-json-normalizer": "^4.2.0",
-    "url-template": "^2.0.8"
+    "url-template": "^3.1.1"
   },
   "devDependencies": {
-    "@babel/plugin-transform-regenerator": "7.16.7",
-    "@babel/plugin-transform-runtime": "7.16.10",
-    "@babel/preset-env": "7.16.11",
-    "@babel/preset-typescript": "7.16.7",
-    "@types/jest": "29.5.12",
     "@typescript-eslint/eslint-plugin": "5.10.1",
     "@typescript-eslint/parser": "5.10.1",
+    "@vitest/coverage-istanbul": "^2.0.5",
+    "@vue/compiler-sfc": "3.3.6",
     "@vue/eslint-config-standard": "6.1.0",
-    "@vue/test-utils": "2.4.1",
-    "axios": "0.25.0",
+    "@vue/test-utils": "2.4.6",
+    "axios": "1.7.4",
```

Comment:
a sneaky update of axios?

Reply:
We don't have this as a peerDependency. But if you want we can leave it at the outdated version.
It is only used for testing and typing purposes.